crawler.py

```diff
@@ -1,15 +1,15 @@
 import asyncio
 import aiohttp
-import json, codecs
+import json
 import datetime
-import os, sys
+import os
+import sys
 import settings
 import requests
 import math
 import collections
 import logging
 import urllib.parse
-import codecs
```

Comment:
I would keep the import codecs line in because we need it for running the unit tests